pages/blog/oss-contributor-workflow.mdx

```diff
@@ -0,0 +1,88 @@
+
+While all this might seem very simple, doing this has a long term benefit of communicating your expectations as a maintainer to the community when they are looking to contribute. Now, this is something which some of the projects tend to ignore.
+
+Then we started thinking about the next important thing which we can address to make both maintainer's and contributor's lives easier - LABELS
+
+As simple as this feature may seem, labels/tags are the most important feature in any project management tool, in this case Github because it gives us the ability to track the progress, the next steps to be done and also communicate the same to various stakeholders, be it the maintainer or the collaborator. Not just this, it can also act as a great way to onboard new collaborators who are interested in contributing to the project.
+
+But you may wonder, labels are an already available feature of Github, what more can we do with it?
+
+The problem in our case is that considering every project had its own repository (as it should be) and everything evolved over time, we had different way of tracking issues in different projects and we followed our own sweet conventions and while this did the job for us when we started off this started getting complicated with more and more projects getting added to our portfolio every month.
+
+This is when we strongly felt the need to sync/orchestrate labels across all the repositories while also respecting the differences between the projects (one size may not fit all). This led us to a proof of concept with different label syncing tools and after quite a bit of analysis, we landed on [Label Syncer](https://github.com/micnncim/action-label-syncer) to manage both master and repository level declarative labels (If someone from Github is reading this, maybe this should be an inbuilt feature 😇)
+
+Doing this was pretty simple. You can find the repo [here](https://github.com/the-guild-org/shared-resources/) which is used as the master repository to push all the labels downstream and we still retain the repo specific labels in their own repository ([like this](https://github.com/Urigo/graphql-mesh/blob/master/.github/labels.yml))
+
+Now that a technical proof of concept was complete, the next question we had was "What labels should we standardize on as the master labels?"
+
+Now, [Urigo](https://twitter.com/UriGoldshtein) had an interesting thought process - since we had already published the contributor workflow guide, why not we label issues keeping the same in mind? All we needed to do was push an issue towards progress from Step 1 to Step N
+
```

Comment:
"The goal here was to make it more obvious for anyone who arrives to an issue where exactly it stands and most importantly, what can you do to help advance the solution"